docs/about/whats-new/whats-new-0.7.0.ipynb

```diff
@@ -0,0 +1,466 @@
+    "\n",
+    "## Python-like shallow/deep copy mechanism\n",
+    "\n",
+    "The most significant change in this scipp release is a fundamental rework of all scipp data structures (variables, data arrays, and datasets).\n",
+    "These now behave mostly like nested Python objects, i.e., sub-objects are shared by default.\n",
+    "Previously there was no sharing mechanism and scipp always made deep-copies.\n",
+    "Some of the effects are exemplified in the following.\n",
+    "\n",
+    "### Variables\n",
+    "\n",
+    "For variables on their own, the new and old implementations mostly yield the same user experience.\n",
+    "Previously, views of variables, such as created when slicing a variable along a dimension, returned a different type &ndash; `VariableView` &ndash; which kept alive the original `Variable`.\n",
+    "This asymmetry is now gone.\n",
+    "Slices or other views of variables are now also of type `Variable`, and all views share ownership of the underlying data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "if not sc.__version__.startswith('0.7'):\n",
+    "    print(f'This notebook was made for scipp-0.7 and will likely not work with your version ({sc.__version__}).')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If a variable refers only to a section of the underlying data buffer this is now indicated in the HTML view as part of the size on the top right, here *\"16 Bytes out of 96 Bytes\"*.\n",
+    "This allows for identification of \"small\" variables that keep alive potentially large buffers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.arange(dim='x', unit='m', start=0, stop=12)\n",
+    "var['x', 4:6]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a variable with sole ownership of a buffer, use the `copy()` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var['x', 4:6].copy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, `copy()` returns a deep copy.\n",
+    "Shallow copies can be made by specifying `deep=False`, which preserves shared ownership of underlying buffers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shallow_copy = var['x', 4:6].copy(deep=False)\n",
+    "shallow_copy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data arrays\n",
+    "\n",
+    "In some cases, extra care must be taken with the new sharing mechanism, just like when working with any other Python library.\n",
```

Comment:
I would maybe swap the lines around, so that it makes more sense that the example you give below is illustrating the extra care that must be taken, rather than illustrating the fixing of some critical issues.
In addition, it would be good if we can sumarise with a few words what the critical issues were.

Reply:
I don't think I would like to document these issues here, since they are gone, and to my knowledge no user ran into them.